WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -310,25 +310,25 @@ open class QuickStartTourGuide: NSObject {
         currentTourState = nil
     }
 
-    static var checklistTours: [QuickStartTour] {
+    static func checklistTours(for blog: Blog) -> [QuickStartTour] {
         return [
             QuickStartCreateTour(),
-            QuickStartViewTour(),
+            QuickStartViewTour(blog: blog),
             QuickStartThemeTour(),
             QuickStartShareTour(),
             QuickStartPublishTour(),
             QuickStartFollowTour()
         ]
     }
 
-    static var customizeListTours: [QuickStartTour] {
+    static func customizeListTours(for blog: Blog) -> [QuickStartTour] {
         return [
             QuickStartCreateTour(),
             QuickStartSiteTitleTour(),
             QuickStartSiteIconTour(),
             QuickStartEditHomepageTour(),
             QuickStartReviewPagesTour(),
-            QuickStartViewTour()
+            QuickStartViewTour(blog: blog)
```

Comment:
`QuickStartSiteTitleTour` also has a waypoint depending on the current blog. It fetches the blog using `WPTabBarController.sharedInstance`

I'm not sure which way is better. I'm leaning towards your new approach. Either way, I think we should unify the approach in the two cases. What do you think?

Reply:
>  Either way, I think we should unify the approach in the two cases. What do you think?

Agreed! Good catch, fixed in 1a7d89d